src/Me/components/List/ListItem.js

```diff
@@ -0,0 +1,70 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+import { ReactComponent as AvailableIcon } from '../../../assets/me/icon-available.svg';
+import { ReactComponent as CountryIcon } from '../../../assets/me/icon-country.svg';
+import { ReactComponent as DescriptionIcon } from '../../../assets/me/icon-description.svg';
+import { ReactComponent as EmailIcon } from '../../../assets/me/icon-email.svg';
+import { ReactComponent as SpokenLanguagesIcon } from '../../../assets/me/icon-spokenLanguages.svg';
+import { ReactComponent as TagsIcon } from '../../../assets/me/icon-tags.svg';
+import { ReactComponent as TitleIcon } from '../../../assets/me/icon-title.svg';
+import { ReactComponent as UnavailableIcon } from '../../../assets/me/icon-unavailable.svg';
+
+// Object w/ keyed SVG Components
+const icons = {
+  email: EmailIcon,
+  spokenLanguages: SpokenLanguagesIcon,
+  country: CountryIcon,
+  title: TitleIcon,
+  tags: TagsIcon,
+  available: AvailableIcon,
+  unavailable: UnavailableIcon,
+  description: DescriptionIcon,
+};
+
+// Styled Components
+const ItemRow = styled.div`
+  display: grid;
+  grid-template-columns: 40px auto;
+  margin-top: 5px;
+`;
+
+const ItemIcon = styled.div`
+  width: 24px;
+  height: 100%;
+  margin-right: 20px;
+`;
+
+const ItemText = styled.div`
+  padding-top: 4px;
+`;
+
+const ListItem = ({ type, value, children }) => {
+  const Icon = icons[type];
+  return (
+    <ItemRow>
+      {Icon && (
+        <ItemIcon data-testid={type}>
+          <Icon />
+        </ItemIcon>
+      )}
+      <ItemText>{value}</ItemText>
+    </ItemRow>
+  );
+};
+
+ListItem.propTypes = {
+  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
+  type: PropTypes.oneOf([
```

Comment:
What if we gonna use `ListItem` in 5 different places with 7 different icons each? Will `ListItem` a list of 35 types?

Reply:
Sorry but I'm not entirely sure I get your point to be honest.

Since we are talking about the `type` prop I'll try to explain the way I say it and hopfuly it will make more sense.

`ListItem` accept a pre defined `type` to achive a few things:
- convinance - it easy to just say the type of icon you want instead of importing it and pass it through. New contributers won't have to worry about it.
- consistancy - minimise the risk of other contributers passing the wrong icons
- easy to maintain - If you need to change an icon you only need to change it once.

So yeah there might be 35 types of icons there might be 1000 types. I don't think its a problem in the context of this project

Reply:
I was thinking about something like that: https://react.semantic-ui.com/elements/list/#types-icon (Click on `List.Item` tab).
It gives more freedom of passing an icon. Also, it might good for code splitting (which we're not doing this right now) so for example if the user is not logged in therefor the back office icons are not needed, we can avoid bundling them in the first load.

Reply:
In the context of semantic ui this is a good approach but in the context of this website if think we can use some abstraction and limitation (for the reason mentioned above).
If code splitting is a requirement then fair enough but I would deem it as over engineering and unnecessary for this component at this stage (and probably later stage as well).

Sorry if i come across as difficult. I'm just expressing my opinion. If you still think it's good idea to take semantic ui approach than i'm happy to make those changes.

Reply:
Don't be :) Indeed there are good points for both of the approaches. We can take this approach and make changes later if needed.

Reply:
So just to be clear, what do yo mean by "this approach"?

Reply:
Yours :)